src/tox/tox_env/api.py

```diff
@@ -219,6 +219,7 @@ def _default_pass_env(self) -> list[str]:
             "LD_LIBRARY_PATH",  # location of libs
             "LDFLAGS",  # linker flags
             "HOME",  # needed for `os.path.expanduser()` on non-Windows systems
+            "PY_COLORS",  # Popular way to force color output in CI
```

Comment:
Use FORCE_COLOR, NO_COLOR instead.

Reply:
Done! Replaced by `FORCE_COLOR` and `NO_COLOR`.